helm/nginx-ingress-controller-app/Configuration.md

```diff
@@ -0,0 +1,14 @@
+# nginx-ingress-controller
+
+This chart installs nginx-ingress-controller and its dependencies as managed applications. An Ingress Controller is a daemon, deployed as a Kubernetes Pod, that watches the apiserver's /ingresses endpoint for updates to the Ingress resource. Its job is to satisfy requests for Ingresses.
+
+
+## Configuration
+
+The following table lists the configurable parameters of the nginx-ingress-controller chart, its dependencies and default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`provider` | `aws`/`azure`/`kvm`. Taken from application catalog configuration. Can't be modified on chart level | `kvm`
+`controller.service.enabled` | If true, create service | `true`
```

Comment:
Why do we let disabled? what are the advantages of not deploy the k8s service?

prometheus for example wont work

Reply:
ok got it now (second review helped)